snap/snapcraft.yaml

```diff
@@ -16,16 +16,27 @@ confinement: strict
 compression: lzo
 
 parts:
-  amber:
+  amber-bash:
     plugin: rust
     source: .
+    build-packages:
+      - yq
     override-pull: |
       craftctl default
-      craftctl set version=$(git describe --tags --abbrev=10)
+      craftctl set version=$(cat $CRAFT_PART_SRC/Cargo.toml | tomlq -rc '.package.version')
+
+  deps:
+    plugin: nil
+    stage-packages:
+      - bc
```

Comment:
this is also irrelevant

Reply:
This is added as dependency even in the README